<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-09T15:30:41 IST" name="Default suite" finished-at="2023-01-09T15:30:52 IST" duration-ms="11174">
    <groups>
    </groups>
    <test started-at="2023-01-09T15:30:41 IST" name="Default test" finished-at="2023-01-09T15:30:52 IST" duration-ms="11174">
      <class name="com.hrm.testcases.VerifyForgotPassword">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrm.testcases.VerifyForgotPassword@28f2a10f]" started-at="2023-01-09T15:30:42 IST" name="setUp" finished-at="2023-01-09T15:30:49 IST" duration-ms="7282" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="passWordRest(java.lang.String)[pri:0, instance:com.hrm.testcases.VerifyForgotPassword@28f2a10f]" started-at="2023-01-09T15:30:49 IST" name="passWordRest" data-provider="ForgotPwd" finished-at="2023-01-09T15:30:51 IST" duration-ms="1905" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[subhash]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Reset Password link sent successfully] but found [test]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Reset Password link sent successfully] but found [test]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.hrm.testcases.VerifyForgotPassword.passWordRest(VerifyForgotPassword.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passWordRest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrm.testcases.VerifyForgotPassword@28f2a10f]" started-at="2023-01-09T15:30:51 IST" name="tearDown" finished-at="2023-01-09T15:30:52 IST" duration-ms="651" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrm.testcases.VerifyForgotPassword -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
